addons/mail/static/src/discuss/call/common/call_invitation.xml

```diff
@@ -2,27 +2,27 @@
 <templates xml:space="preserve">
 
     <t t-name="discuss.CallInvitation">
-        <div class="o-discuss-CallInvitation d-flex flex-column m-2 p-5 border border-dark rounded-1 text-bg-900" t-attf-class="{{ className }}" t-ref="root">
-            <div t-if="props.thread.rtcInvitingSession" class="o-discuss-CallInvitation-correspondent d-flex flex-column justify-content-around align-items-center text-nowrap">
+        <div class="o-discuss-CallInvitation d-flex flex-column m-2 border border-dark rounded-1 text-bg-900" t-attf-class="{{ className }}" t-ref="root">
+            <div t-if="props.thread.rtcInvitingSession" class="o-discuss-CallInvitation-correspondent m-5 d-flex flex-column justify-content-around align-items-center text-nowrap">
                 <img class="mb-2 rounded-circle cursor-pointer o_object_fit_cover"
                     t-att-src="props.thread.rtcInvitingSession.channel_member_id.persona.avatarUrl"
                     t-on-click="onClickAvatar"
                     alt="Avatar"/>
                 <span class="w-100 fw-bolder text-truncate text-center overflow-hidden" t-esc="props.thread.rtcInvitingSession.channel_member_id.persona.name"/>
                 <span class="fst-italic opacity-75">Incoming Call...</span>
             </div>
-            <div class="d-flex justify-content-around align-items-center w-100 mt-4">
-                <button class="btn user-select-none p-2 rounded-circle border-0 btn-danger"
+            <div class="d-flex justify-content-around m-1 gap-1 align-items-center justify-content-between">
+                <button class="btn user-select-none flex-grow-1 p-2 border-0 btn-danger"
```

Comment:
use `t-set` to reduce the duplicated `btn user-select-none flex-grow-1 p-2 border-0` and the style can be garanteed the same